Kernel/Arch/x86/ProcessorInfo.h

```diff
@@ -32,7 +33,7 @@ class ProcessorInfo {
 
     const String& cpuid() const { return m_cpuid; }
     const String& brandstr() const { return m_brandstr; }
-    const String& features() const { return m_features; }
+    StringView features() const { return m_features->view(); }
```

Comment:
Would it also make sense to convert the functions above this to `StringView`? `String const&` with the double-indirection seems generally undesirable to me.

Reply:
Yes, but that's for another day.